graphql/task_resolver.go

```diff
@@ -509,6 +509,43 @@ func (r *taskResolver) Status(ctx context.Context, obj *restModel.APITask) (stri
 	return *obj.DisplayStatus, nil
 }
 
+// TaskFiles is the resolver for the taskFiles field.
+func (r *taskResolver) TaskFiles(ctx context.Context, obj *restModel.APITask) (*TaskFiles, error) {
```

Comment:
do we have plans to remove the other taskfiles function in query so we dont have the same thing twice?

Reply:
yessir I'll do it within this ticket 🫡